SampleApp/Podfile

```diff
@@ -1,6 +1,9 @@
 use_frameworks!
 
-target :ConsistencyManagerDemo do
-  pod 'ConsistencyManager', :path => '..'
+target 'ConsistencyManagerDemo' do
+	pod 'ConsistencyManager', :path => '..'
```

Comment:
nitpick - this is now a tab instead of spaces

originally it was two spaces, but the target below has 4 spaces - should be consistent